corehq/middleware.py

```diff
@@ -257,16 +259,24 @@ def __init__(self, get_response=None):
             re.compile(regex.format(domain=legacy_domain_re)) for regex in regexes
         ]
 
-    def _bypass_sessions(self, request, domain):
-        return (toggles.BYPASS_SESSIONS.enabled(domain) and
+    def _bypass_sessions(self, request):
+        return (toggles.BYPASS_SESSIONS.enabled(uuid.uuid4().hex, toggles.NAMESPACE_OTHER) and
             any(rx.match(request.path_info) for rx in self.bypass_re))
```

Comment:
nit: I'd put the regex tests first so we don't hit the cache unnecessarily. The regex tests should be very fast too.

Sorry only saw this now.

Reply:
Good point, I've swapped the conditions. Though, in this case, since almost all are mobile hits, they will all pass the regex, so it's not orders of magnitude difference.